### PR TITLE
august heartbeat

### PR DESCRIPTION
**Pull request by jendefig**
Here's the August Heartbeat.  I'd like this to go out today/tonight if possible because I'd like to include it in tomorrow's monthly email.  

Need help with Docs updates.  Per @daavoo's excellent suggestion, I will be incorporating Doc updates monthly in the Heartbeat.  Let me know how I should edit this section.  There are probably important things from the last couple months that I should include and haven't.   My plan for the Twitter swag grab is to ask people to put the doc updates in the comments. 😁 🤫 

Also please help with the section about @isidentical's project.  I do not understand what this is, so I don't know how to make this better.  😬

### CODE REVIEW

File: content/blog/2021-08-16-august-21-dvc-heartbeat.md
Comment (isidentical) [inline content/blog/2021-08-16-august-21-dvc-heartbeat.md:156]: I think this fits better, also thanks for mentioning @jendefig!
```suggestion
A python refactoring tool, [refactor](https://github.com/isidentical/refactor), created by team
```

File: content/blog/2021-08-16-august-21-dvc-heartbeat.md
Comment (iesahin) [inline content/blog/2021-08-16-august-21-dvc-heartbeat.md:111]: MLOPs -> MLOps

File: content/blog/2021-08-16-august-21-dvc-heartbeat.md
Comment (iesahin) [inline content/blog/2021-08-16-august-21-dvc-heartbeat.md:138]: It's better to give my https://emresahin.net link here. I don't use that Linked.in page.

**Review by iesahin — CHANGES_REQUESTED** [review]
Thank you @jendefig Great stuff as always 💯

File: content/blog/2021-08-16-august-21-dvc-heartbeat.md
Comment (jorgeorpinel) [inline content/blog/2021-08-16-august-21-dvc-heartbeat.md:133]: Can you remind us the rationale behind sharing docs updates via blog? I'm not opposed but I'd like to discuss the scope with the team. The current list below is more detailed than what we even share internally with the company 🙂

Comment (daavoo) [reply content/blog/2021-08-16-august-21-dvc-heartbeat.md:133]: I originally suggested that we should communicate somehow the docs improvements being added. However I don't have a strong opinion on where (hearbeat, blog post, other) or how to aggregate the updates.

Although I agree partially with:

> all content should organically reach readers via search engines, support channels, etc

I think that docs should showcase/promote their updates similar to how releases work for software (i.e. [DVC](https://github.com/iterative/dvc/releases)/[CML](https://github.com/iterative/cml/releases)), where users can see a summary of the changes being added recently. 

I don't think it's realistic to expect that an user looking for info will be periodically re-searching the docs for updates or re-asking in support channels. In my experience, when you have a need, you look for info at one point in time and just deal with what it is published at that moment.

A real example:  

Last week I was looking for info regarding sharing experiments. I found a [section in Getting Started](https://dvc.org/doc/start/experiments#sharing-experiments) and then just the command reference. I used that succinct info and tried things by myself. My life would have been easier if I had access to the [Sharing Experiments User Guide](https://dvc.org/doc/start/experiments#sharing-experiments), which got merged a few days before my initial search.

I just notice that new guide because I was contributing to `dvc.org` and it happened to be an Open P.R. at the same time I checked the P.Rs. page. 

If I were an external user, how would I noticed this change that helped me?

Comment (jendefig) [reply content/blog/2021-08-16-august-21-dvc-heartbeat.md:133]: > I don't think it's realistic to expect that an user looking for info will be periodically re-searching the docs for updates or re-asking in support channels. In my experience, when you have a need, you look for info at one point in time and just deal with what it is published at that moment.

I agree with this.  And so it could be helpful to point out some of our changes.  My list may be more detailed - that's why I need help with what should be here.

I've heard now repeatedly from our Community that it seems people get things working for them and then may not check back. Even our champions that love us and use our tools: João - didn't know about Studio functionality, Sami and not using the experiments, I'm forgetting who, but someone using DVC, but yet to implement CML, though they want to.  All these people are busy making things right in their own teams.  To give them a monthly update on things that *may make their lives easier* is a very good idea.  (to the extent that they actually read Heartbeat - but adding this info may help drive more Heartbeat views)

Comment (jendefig) [reply content/blog/2021-08-16-august-21-dvc-heartbeat.md:133]: @jorgeorpinel I also think where Community members contribute significantly to docs changes we should shout that out too which is why a included @ meirale.  This will a.)recognize their contribution and b.) encourage more contribution.

File: content/blog/2021-08-16-august-21-dvc-heartbeat.md
Comment (jorgeorpinel) [inline content/blog/2021-08-16-august-21-dvc-heartbeat.md]:
```suggestion
- Complete [DVCLive docs](https://dvc.org/doc/dvclive)
- We have a new [Glossary page](https://dvc.org/doc/user-guide/glossary) and a
  fist Basic Concepts page
  ([_DVC Wokspace_](https://dvc.org/doc/user-guide/basic-concepts/workspace))
- Added videos to [Get Started pages](https://dvc.org/doc/start) and
  [Checkpoints guide](https://dvc.org/doc/user-guide/experiment-management/checkpoints)
- Authentication examples for
  [Azure Blob remote storage](https://dvc.org/doc/command-reference/remote/modify)
  from Community member @meierale ❤️ 
- CML docs migration to [CML.Dev](https://cml.dev/doc)
```

> Let's leave the Experiments Guide expansions (ongoing work and reviews) for next month? 😉

**Review by jorgeorpinel — APPROVED** [review]
Suggestions to make this mergeable. We can continue to discuss on docs updates scope after.

**Comment by jendefig** [comment]
@jorgeorpinel @shcheklein I don't know what's going on with the links check.  They are correct and go where they are supposed to. 🤷🏻‍♀️ This is ready to merge.

File: content/blog/2021-08-17-august-21-dvc-heartbeat.md
Comment (jorgeorpinel) [inline content/blog/2021-08-17-august-21-dvc-heartbeat.md:47]: This one

File: content/blog/2021-08-17-august-21-dvc-heartbeat.md
Comment (jorgeorpinel) [inline content/blog/2021-08-17-august-21-dvc-heartbeat.md:227]: And this one.

Comment (jorgeorpinel) [reply content/blog/2021-08-17-august-21-dvc-heartbeat.md:227]: But I see they both render properly in https://dvc-org-august-heartbea-moy8x7.herokuapp.com/blog/august-21-dvc-heartbeat so yeah we're good to merge here.

**Review by jorgeorpinel — COMMENTED** [review]
> I don't know what's going on with the links check. They are correct

@jendefig The check's name is misleading. It checks all URLs, not just links. In this case it's complaining about 2 images:

File: content/blog/2021-08-17-august-21-dvc-heartbeat.md
Comment (jorgeorpinel) [reply content/blog/2021-08-17-august-21-dvc-heartbeat.md:47]: Before I merge, let me post here what the actual URL paths are in the rendered page for the record.
For this one it's

/static/6cfa523455e454fb01e9f7fabb1cf96f/fa73e/lj-miranda-data-centric.webp

File: content/blog/2021-08-17-august-21-dvc-heartbeat.md
Comment (jorgeorpinel) [reply content/blog/2021-08-17-august-21-dvc-heartbeat.md:227]: The actual URL rendered for this one is

/static/83fdf48f7311c67c558afe07fa5a639b/fa73e/survey.webp

Comment (jorgeorpinel) [reply content/blog/2021-08-17-august-21-dvc-heartbeat.md:227]: BTW what is this /uploads -> /static change? I suspect @rogermparent and/or @casperdcl may know. We need to update the link checker to support this.

File: content/blog/2021-08-16-august-21-dvc-heartbeat.md
Comment (jorgeorpinel) [reply content/blog/2021-08-16-august-21-dvc-heartbeat.md]: > I don't think it's realistic to expect that an user looking for info will be periodically re-searching the docs

@daavoo @jendefig Then it' also not realistic that they'll look in the blog for the same info.

Writing/reviewing this takes some effort so I think it's best to have a clear reason or problem we're solving here.

The question of how many website users are docs regulars and for what reasons they come back is interesting (I would assume mainly they return to the cmd ref.) but again it's not clear to me how we would serve them with these updates. Do they even overlap with blog readers? We just don't have all this info.

I think that in blog updates we should actual highlight product updates instead, and link to the new/improved docs that usually accompany those product releases. That (which we already do probably) feels more natural to me.

> to the extent that they actually read Heartbeat

Exactly. So maybe it may be also a matter of having a social-media friendly summary of heartbeats to post on Discord/Twitter along with a link to the full article. But that's up to @iterative/devrel I think.

Comment (jorgeorpinel) [reply content/blog/2021-08-16-august-21-dvc-heartbeat.md]: > where Community members contribute significantly to docs changes we should shout that out too which is why a included @ meirale

I like that idea but it could just be those docs updates we include in the heartbeat as part of the community section. BTW how will that person know he was mentioned?

**Comment by jorgeorpinel** [comment]
⚠️ dayum looks like this didn't build. Error log:

```
       Uploading ".cache" to s3://.../...-prod-cache/
       ".cache" uploaded in: 37.430s
       "public" uploaded in: 52.502s
       Cleaning all local cache!
Error: CloudFlare cache clear request returned non-ok status 403: Forbidden
    at module.exports (/tmp/build_b1fb9cb4/scripts/clear-cloudflare-cache.js:52:11)
    at runMicrotasks (<anonymous>)
    at processTicksAndRejections (node:internal/process/task_queues:94:5)
    at async main (/tmp/build_b1fb9cb4/scripts/deploy-with-s3.js:136:5)
error Command failed with exit code 1.
       info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
-----> Build failed
```

Yelp @rogermparent @julieg18 is this maybe related to the images mentioned under https://github.com/iterative/dvc.org/pull/2723#pullrequestreview-732328532 above?

File: content/blog/2021-08-16-august-21-dvc-heartbeat.md
Comment (jorgeorpinel) [reply content/blog/2021-08-16-august-21-dvc-heartbeat.md]: p.s. guys feel free to include in a [minute](https://www.notion.so/iterative/Minutes-35878778da26403ab78e08e783d2d58a) and join a docs meeting when you can if you'd like to discuss this with the whole team 🙂

Comment (jorgeorpinel) [reply content/blog/2021-08-16-august-21-dvc-heartbeat.md]: Quick recap on this: I believe we agreed to mark certain docs updates in our internal reports as "may be worth sharing with the community" and let #devrel take it from there. Thanks

File: content/blog/2021-08-17-august-21-dvc-heartbeat.md
Comment (rogermparent) [reply content/blog/2021-08-17-august-21-dvc-heartbeat.md:227]: > BTW what is this /uploads -> /static thingy? I suspect @rogermparent and/or @casperdcl may know.

I believe this is due to Gatsby image processing, since the resulting imageset isn't at the same path as the original.

> We need to update the link checker to support or ignore this.

Yeah, probably, ignoring all MD image URLs would be the most obvious solution, and I don't think there doing so would clobber anything since we process all MD images now.

Comment (jorgeorpinel) [reply content/blog/2021-08-17-august-21-dvc-heartbeat.md:227]: https://github.com/iterative/link-check/issues/9

**Pull request merged**
